Makefile.dune

```diff
@@ -34,10 +34,10 @@ watch: voboot
 release: voboot
 	dune build $(DUNEOPT) -p coq
 
-apidoc:
+apidoc: voboot
         # Ugly workaround for https://github.com/ocaml/odoc/issues/148
 	mv checker/dune checker/dune.disabled || true
-	dune build $(DUNEOPT) @doc
+	dune build $(DUNEOPT) @doc || mv checker/dune.disabled checker/dune
```

Comment:
Add some exit 1 so it'll fail if there's a problem.

Reply:
Done thanks, I hope I got the precedence right.